tools/pd-ctl/tests/scheduler/scheduler_test.go

```diff
@@ -483,7 +483,8 @@ func (suite *schedulerTestSuite) checkScheduler(cluster *pdTests.TestCluster) {
 		pdTests.MustPutStore(re, cluster, store)
 	}
 	re.Equal("5.2.0", leaderServer.GetClusterVersion().String())
-	// After upgrading, we should not use query.
+	// After upgrading, we can use query.
+	expected1["write-leader-priorities"] = []any{"query", "byte"}
```

Comment:
Why changed?

Reply:
It seems that after 5.2.0 defaultPrioritiesConfig is the query and byte.

https://github.com/tikv/pd/blob/cd0ffbafaa1adf9b3bc4d8cbe90e993c79f99e2b/pkg/schedule/schedulers/hot_region_config.go#L47